modules/serverless-minimal-requirements-serving.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-minimal-requirements-serving_{context}"]
+= Minimal requirements of OpenShift Serverless Serving
+
+While the default set-up is suitable for medium-sized workloads, this might be over-sized for smaller set-ups or under-sized for high-workload scenarios.
+To configure OpenShift Serverless Serving for a minimal workload scenario, you need to know the idle consumption of the system components.
+
+[id="serverless-minimal-requirements-serving-idle-consumption_{context}"]
+== Idle consumption
+
+The idle consumption is dependent on the number of Knative Services. The following memory utilization has been measured for the components in the `knative-serving` and `knative-serving-ingress` OpenShift projects:
+
+* 170 Knative Services
```

Comment:
This looks like it mixes two separate measurements.  If this is about *idle* behavior with existence of 0/100/500/1000 idle ksvcs, than this list doesn't make sense and should be removed.